package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-scm-base",
-  "version": "2.4.0",
+  "version": "2.5.0",
```

Comment:
I know we did that in the past to make sure we were pulling in the right version in scm-github and scm-bitbucket. But we could probably pin to a patch version